docs/source/tutorial/04-execute-the-query.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: "4. Execute the query"
+---
+
+To execute the query, you will need to create an `ApolloClient` and make a new call with your query.
+
+## Create an ApolloClient
+
+Create a new file named Apollo.kt in the `com.example.rocketreserver` package and add the ApolloClient:
+
+```kotlin:title=app/src/main/java/com/example/rocketreserver/Apollo.kt
+val apolloClient = ApolloClient.builder()
+    .serverUrl("https://apollo-fullstack-tutorial.herokuapp.com")
+    .build()
+```
+
+`apolloClient` is a top-level variable so that it can be reused from other parts of the app later on.
```

Comment:
maybe to mention that we recommend to do so, similar to 
```
OkHttp performs best when you create a single OkHttpClient instance and reuse it for all of your HTTP calls. This is because each client holds its own connection pool and thread pools. Reusing connections and threads reduces latency and saves memory. Conversely, creating a client for each request wastes resources on idle pools.
```